src/WebJobs.Script/Http/DefaultHttpProxyService.cs

```diff
@@ -21,6 +21,7 @@ internal class DefaultHttpProxyService : IHttpProxyService, IDisposable
         private readonly HttpMessageInvoker _messageInvoker;
         private readonly ForwarderRequestConfig _forwarderRequestConfig;
         private readonly ILogger<DefaultHttpProxyService> _logger;
+        private readonly HttpTransformer _httpTransformer;
```

Comment:
nit: you can inline this.

```suggestion
        private readonly HttpTransformer _httpTransformer = new ScriptInvocationRequestTransformer();
```